frontend/src/pages/org/collections-list.ts

```diff
@@ -494,16 +475,45 @@ export class CollectionsList extends LiteElement {
               ? msg("1 page")
               : msg(str`${this.numberFormatter.format(col.pageCount)} pages`)}
           </div>
+          <div
+            class="col-span-1 truncate text-xs text-neutral-500 font-monostyle"
+          >
+            ${col.publishedUrl
+              ? html`
+                  <a
+                    class="text-primary hover:text-indigo-500"
+                    href="https://replayweb.page?source=${new URL(
+                      col.publishedUrl, window.location.href
+                    ).href}"
+                    target="_blank"
+                    rel="noopener noreferrer nofollow"
+                    @click=${(e: MouseEvent) => {
+                      e.stopPropagation();
+                    }}
+                  >
+                    <sl-icon
+                      class="inline-block align-middle"
+                      name="link-45deg"
+                    ></sl-icon>
+                    <span class="inline-block align-middle"
+                      >${msg("View")}</span
+                    >
+                  </a>
+                `
+              : html`<span role="presentation">--</span>`}
+          </div>
           <div
             class="actionsCol absolute top-0 right-0 md:relative col-span-1 flex items-center justify-center"
           >
             ${this.isCrawler ? this.renderActions(col) : ""}
           </div>
         </div>
-      </a>
+      </div>
     </li>`;
 
   private renderActions = (col: Collection) => {
+    // FIXME replace auth token post-workshop
+    const authToken = this.authState!.headers.Authorization.split(" ")[1];
```

Comment:
@SuaYoo is this something we still need to fix? This might be ok for now, since its a one-time download link for logged-in user.